CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2220,6 +2220,150 @@ private void OnMouseButton(MouseButtonEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Provides class handling for the <see cref="E:System.Windows.TouchDown" /> routed event that occurs when a touch presses inside this element.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.TouchEventArgs" /> that contains the event data.
+        protected override void OnTouchDown(TouchEventArgs e)
+        {
+            Focus();
+            CaptureTouch(e.TouchDevice);
```

Comment:
Please add a code comment explaining the reason for the capture, what use case is it required for.

Reply:
Added comments explaining my reasoning for the Captures.